src/boot/efi/efi-string.h

```diff
@@ -124,8 +153,16 @@ static inline void *mempcpy(void * restrict dest, const void * restrict src, siz
         memcpy(dest, src, n);
         return (uint8_t *) dest + n;
 }
+
+static inline void explicit_bzero_safe(void *bytes, size_t len) {
```

Comment:
This shouldn't be here, because we now have 3f92dc2fd4070b213e6bc85263a9bef06ec9a486 (backported as ffff09335760efc655faf093b6fbb364cfae4ad7 in the other large PR) in RHEL 9 that drops this chunk.